tests/test_api.py

```diff
@@ -657,10 +657,10 @@ def test_if_remove_with_auto_commit_then_invoke_stash_and_commit(
             gto.api.remove(repo=repo.working_dir, name=name, commit=True)
 
     mocked_stashed_changes.assert_called_once_with(
-        repo=repo.working_dir, include_untracked=True
+        repo=git.Repo(repo.working_dir), include_untracked=True
```

Comment:
this should work I assume
```suggestion
        repo=repo, include_untracked=True
```

Reply:
This test is already working

Reply:
yes, I meant, it should be faster)

Reply:
oh that's what you meant, that's right